lua/starfall/libs_sv/constraint.lua

```diff
@@ -113,6 +113,17 @@ function constraint_library.weld(e1, e2, bone1, bone2, force_lim, nocollide)
 end
 
 --- Axis two entities
+-- @param e1 The first entity
+-- @param e2 The second entity
+-- @param bone1 Number bone of the first entity
+-- @param bone2 Number bone of the second entity
+-- @param v1 Position on the first entity, in its local space coordinates
+-- @param v2 Position on the second entity, in its local space coordinates
+-- @param force_lim Amount of force until it breaks, 0 = Unbreakable
+-- @param torque_lim Amount of torque until it breaks, 0 = Unbreakable
+-- @param friction Friction of the constraint
+-- @param nocollide Bool whether or not to nocollide the two entities
+-- @param laxis Local axis, if set, then position on the second entity (v2 argument) will not be used in the final constraint, however it's still required
```

Comment:
This sets the axis normal in the coordinates of the first entity I think. Need to verify

Reply:
You're correct, i will additionally simplify the description cuz it's a mess